HMCL/src/main/resources/assets/lang/I18N.properties

```
@@ -1412,6 +1412,74 @@ sponsor.hmcl=Hello Minecraft! Launcher is a FOSS Minecraft launcher that allows
 system.architecture=Architecture
 system.operating_system=Operating System
 
+terracotta=Multiplayer
+terracotta.easytier=About EasyTier
+terracotta.terracotta=Terracotta | Multiplayer
+terracotta.status=Lobby
+terracotta.back=Exit
+terracotta.network_warning=Terracotta | Multiplayer is based on P2P. The final experience depends greatly on your network.
+terracotta.sudo_installing=HMCL must verify your password before installing Multiplayer Core
+terracotta.from_local.title=Third-party download channels for Multiplayer Core
+terracotta.from_local.desc=In some areas, the built-in default download channel may be unstable.
+terracotta.from_local.guide=Please download Multiplayer Core package named %s. Once downloaded, drag the file into the current page to install it.
+terracotta.from_local.file_name_mismatch=You should download the Multiplayer Core package named %1$s instead of %2$s
+terracotta.status.bootstrap=Gathering information
+terracotta.status.uninitialized.not_exist=Multiplayer Core: Not Downloaded
+terracotta.status.uninitialized.not_exist.title=Download Multiplayer Core (~ 8MiB)
+terracotta.status.uninitialized.update=Multiplayer Core: Update Available
+terracotta.status.uninitialized.update.title=Update Multiplayer Core (~ 8MiB)
+terracotta.status.uninitialized.desc=You legally promise to strictly abide by all laws and regulations of your country or region during the multiplayer process.
+terracotta.confirm.title=User Notice
+terracotta.confirm.desc=Multiplayer is based on P2P; the final experience depends greatly on your network. You legally promise to strictly abide by all laws and regulations of your country or region during the multiplayer process.
+terracotta.status.preparing=Multiplayer Core: Downloading (DO NOT exit HMCL)
+terracotta.status.launching=Multiplayer Core: Initializing
+terracotta.status.unknown=Multiplayer Core: Initializing
+terracotta.status.waiting=Multiplayer Core: Ready
+terracotta.status.waiting.host.title=I want to host a session
+terracotta.status.waiting.host.desc=Create a room and generate an invite code to play with friends
+terracotta.status.waiting.host.launch.title=You seem to have forgotten to start the game
+terracotta.status.waiting.host.launch.desc=No running game found
+terracotta.status.waiting.host.launch.skip=Game has started
+terracotta.status.waiting.guest.title=I want to join a session
+terracotta.status.waiting.guest.desc=Enter the invite code from the host player to join the game world
+terracotta.status.waiting.guest.prompt.title=Please enter the invite code from the host
+terracotta.status.waiting.guest.prompt.invalid=Invalid invite code
+terracotta.status.scanning=Scanning LAN worlds
+terracotta.status.scanning.desc=Please <a href="hmcl://game/launch">start the game</a>, open a save, press ESC, select Open to LAN, and create a LAN world.
```

1. Maybe we should emphasize the word *world* that appears in the Minecraft UI to make it easier for general players to understand.
2. Follow the description in Minecraft.

```suggestion
terracotta.status.scanning.desc=Please <a href="hmcl://game/launch">start the game</a>, open a world, press ESC, select "Open to LAN", then select "Start LAN World".
```